client/src/components/SetVacations.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <v-container>
+    <v-form ref="form" @submit.prevent="setVacations">
+      <v-row class="justify-center align-center">
+        <v-col cols="12">
+          <v-alert
+            class="mb-3"
+            type="info"
+            text="The provided figures reflect the current year's details for this office. Make sure to change the year if you want to set new data. Otherwise, the existing 2024 data will be amended."
+          >
+          </v-alert>
+          <v-alert class="mb-3" type="info" text="You need to set the data annually."></v-alert>
+          <v-text-field
+            v-model="office_balance.year"
+            type="number"
+            label="Year"
+            :rules="requiredRules"
+          ></v-text-field>
+          <v-text-field
+            v-model="office_balance.annual_leaves"
+            type="number"
+            label="Annual Leaves"
+            :rules="requiredRules"
+          ></v-text-field>
+          <v-text-field
+            v-model="office_balance.leave_excuses"
+            type="number"
+            label="Leave Excuses"
+            :rules="requiredRules"
+          ></v-text-field>
+          <v-text-field
+            v-model="office_balance.emergency_leaves"
+            type="number"
+            label="Emergency Leaves"
+            :rules="requiredRules"
+          ></v-text-field>
+          <v-text-field
+            v-model="formattedDates"
+            label="Public Holidays"
+            placeholder="Select or enter dates separated by commas"
+            @click="toggleDatePicker"
+            :rules="requiredRules"
+            readonly
+          >
+            <template v-slot:append-inner>
+              <v-icon>mdi-calendar</v-icon>
+            </template>
+          </v-text-field>
+
+          <v-date-picker
+            v-if="datePickerVisible"
+            v-model="selectedDates"
+            @click:prev="preventDefault"
+            @click:next="preventDefault"
```

Comment:
use .prevent instead

```suggestion
            @click:prev.prevent
            @click:next.prevent
```